redbot/setup.py

```diff
@@ -401,7 +410,15 @@ async def remove_instance_interaction() -> None:
         "Note: Choosing PostgreSQL will prevent the setup from being completely non-interactive."
     ),
 )
-@click.option("--overwrite-existing-instance", type=bool, is_flag=True)
+@click.option(
+    "--overwrite-existing-instance",
+    type=bool,
+    is_flag=True,
+    help=(
+        "Overwrites an existing instance.\n",
+        "Note: Using this can erase all the data related to the existing instance."
```

Comment:
```suggestion
        "Note: Using this can erase all data related to the existing instance."
```

Reply:
```suggestion
        "Note: Using this can erase all the data related to the existing instance.",
```
Fix for failing style check I noticed as well.